gradle/javaToolchain.gradle

```diff
@@ -13,18 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-final Integer DEFAULT_JDK_VERSION = 11
+final Integer DEFAULT_JDK_VERSION = 17
 
 def javaMainProjects = subprojects - project(":util:taglets")
 
 allprojects {
 
     def javaVersion = (findProperty("javaVersion") ?: DEFAULT_JDK_VERSION.toString()).toInteger()
     project.pluginManager.withPlugin("java") {
-        logger.info("Compiling ${project.name} using JDK${DEFAULT_JDK_VERSION}")
-        java {
-            toolchain {
-                languageVersion = JavaLanguageVersion.of(DEFAULT_JDK_VERSION)
+        if (!project.pluginManager.hasPlugin("scala")) {
```

Comment:
Don't force java version for Scala projects.  Scala 2.11 doesn't support Java 17. It EOL'd in 2017 - its worth determining our support plan for older scala versions (2.11 & 2.12)

Reply:
How does the underlying JVM get chosen when we specify the Scala version?

Reply:
I think its via the java toolchain support mechanism.